flex-checks/src/test/resources/checks/NonEmptyCaseWithoutBreak.as

```diff
@@ -15,7 +26,15 @@ function foo() {
     case 0:
     case 1: // NOK
       doSomething();
+    case 2: // NOK
+      if (true) {
+        break;
+      }
+    case 3: // NOK
+      break;
+      doSomething();
     default: // NOK
```

Comment:
@pynicolas Indeed, according to RSPEC this line should be compliant. @lindamartin do you agree?
One more case different from RSPEC-128 - "Use of continue statement", nothing about "continue" in rule implementation. I'm going to add this case either